src/Orchard.Web/Modules/Orchard.Templates/Views/DefinitionTemplates/Parts.Shape.cshtml

```diff
@@ -14,13 +14,6 @@
 
 <fieldset>
     <div>
-        @Html.LabelFor(m => m.Name, T("Name"))
-        @Html.TextBoxFor(m => m.Name, new { @class = "text medium" })
-        <span class="hint">@T("The name of this template, which can be used to reference from code (as when using shape type names) and tokens.")</span>
-    </div>
-</fieldset>
-<fieldset>
-    <div>
```

Comment:
I don't think this is correct, although I also see build errors with compilation from this piece of code - I'll look into it.